lib/monitoring/metrics/api_exception_counter.rb

```diff
@@ -0,0 +1,29 @@
+module Monitoring
+  module Metrics
+    class ApiExceptionCounter
+      attr_reader :registry, :pubsub, :metric_name, :docstring, :labels, :sub_event_name
+
+      def setup(registry, pubsub)
+        @registry = registry
+        @pubsub = pubsub
+        @metric_name = :conjur_http_server_request_exceptions_total
+        @docstring = 'The total number of API exceptions raised by Conjur.'
+        @labels = %i[operation exception message]
+        @sub_event_name = 'conjur.request_exception'
+
+        # Create/register the metric
+        Metrics.create_metric(self, :counter)
+      end
+
+      def update(payload)
+        metric = @registry.get(@metric_name)
```

Comment:
To fix the `assumes too much for instance variable '@metric_name'` CodeClimate warning, I believe you can refer to the method:

```ruby
 metric = @registry.get(metric_name)
```